js/main.js

```diff
@@ -1,2 +1,32 @@
 /* global data */
 /* exported data */
+
+function handleInput(event) {
+  $image.src = event.target.value;
+}
+
+function handleSubmit(event) {
+  event.preventDefault();
+  var newEntry = {};
+  var newObj = {
+    title: $entryForm.elements.title.value,
+    imageURL: $entryForm.elements.photourl.value,
+    notes: $entryForm.elements.notes.value
+  };
+  for (var i in newObj) {
+    newEntry[i] = newObj[i];
+  }
```

Comment:
What is the purpose of this? It looks like `newObj` is doing the job just fine, you don't need to immediately copy that object to another object.